src/WebJobs.Script/Workers/Rpc/SharedMemoryDataTransfer/SharedMemoryManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
```

Comment:
Move contents from folder: `SharedMemoryDataTransfer` up one level - under workers - as this can used by non-Rpc workers as well.